assets/javascripts/discourse/initializers/chat-setup.js

```diff
@@ -124,6 +127,156 @@ export default {
           }
         });
       });
+      api.addSidebarSection(
+        (BaseCustomSidebarSection, BaseCustomSidebarSectionLink) => {
+          const SidebarChatSectionLink = class extends BaseCustomSidebarSectionLink {
+            constructor({ channel }) {
+              super(...arguments);
+              this.channel = channel;
+            }
+
+            get name() {
+              return this.channel.chatable_id;
+            }
+
+            get route() {
+              return "chat.channel";
+            }
+
+            get model() {
+              return {
+                chatChannel: this.channel,
+                channelId: this.channel.id,
+                channelTitle: this.channel.chatable.slug,
```

Comment:
where is this used?

Reply:
This is that one -> https://github.com/discourse/discourse-chat/pull/977#discussion_r919512212